src/InterestPayout.Common/Domain/AssetInterest.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+
+namespace InterestPayout.Common.Domain
+{
+    public class AssetInterest
+    {
+        public long Id { get; }
+        
+        public string AssetId { get; }
+        
+        public decimal InterestRate { get; }
+        
+        public DateTimeOffset ValidUntil { get; }
```

Comment:
Usually we use DateTime at domain layer because all server-side time is expressed in UTC. We are enforced to use DateTimeOffset at persistence layer because using of DateTime leads to incorrect conversion at gRPC layer

Reply:
`+`